ci_cd/tasks/api_reference_docs.py

```diff
@@ -149,7 +151,7 @@ def write_file(full_path: Path, content: str) -> None:
 
     if pre_commit:
         # Ensure git is installed
-        result: "Result" = context.run("git --version", hide=True)
+        result: Result = context.run("git --version", hide=True)
```

Comment:
Codecov warning. But this probably does not need to be tested (?).

Reply:
Not relevant any longer.